cni/log/logger.go

```diff
@@ -46,6 +47,7 @@ func newFileLogger(cfg *Config) *zap.Logger {
 
 	core := zapcore.NewCore(jsonEncoder, logFileWriter, logLevel)
 	Logger = zap.New(core)
+	Logger = Logger.With(zap.Int("pid", os.Getpid()))
 
-	return Logger.Named(cfg.Name)
+	return Logger.Named("")
```

Comment:
We don't need to return the Logger.Named(""). We can now just return the `Logger`. By default Logger is unnamed.